src/test-integration/java/com/bettercloud/vault/util/VaultContainer.java

```diff
@@ -193,10 +174,11 @@ public void setupBackendPki() throws IOException, InterruptedException {
      *
      * @throws IOException
      * @throws InterruptedException
+     * @param cert
      */
-    public void setupBackendCert() throws IOException, InterruptedException {
+    public void setupBackendCert(String cert) throws IOException, InterruptedException {
         runCommand("vault", "login", "-ca-cert=" + CONTAINER_CERT_PEMFILE, rootToken);
-
+        runCommand("sh", "-c", "cat <<EOL >> " + CONTAINER_CLIENT_CERT_PEMFILE + "\n" + cert + "\nEOL");
```

Comment:
Running on CI against Travis or Github it seems the vault container's uid/gid is the only one with permissions to this file.

Reply:
@steve-perkins You would recommend Travis CI for this repo once this PR is merged.
Otherwise, GitHub Action CI will soon be generally available.

> [GitHub Actions is generally available on November 13](https://github.blog/2019-08-08-github-actions-now-supports-ci-cd/)

Reply:
Current CI runs are both available here: https://github.com/casz/vault-java-driver/runs/202654214